test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/Implementation/ExportClient/OtlpHttpTraceExportClientTests.cs

```diff
@@ -125,12 +125,13 @@ public void SendExportRequest_ExportTraceServiceRequest_SendsCorrectHttpRequest(
 
         void RunTest(Batch<Activity> batch)
         {
+            var deadlineUtc = DateTime.UtcNow.AddMilliseconds(options.TimeoutMilliseconds);
```

Comment:
It might not affect test or benchmarks but we should either use the correct timeout which is `options.HttpClientFactory().Timeout` or we should add a comment in the tests saying the timeout value provided does not matter for the test/benchmark.